### ISSUE TITLE
MySQL configuration

### ISSUE THREAD

**Issue by hackery** [spring-petclinic/spring-petclinic-angularjs#24]
The main README includes this comment (inherited from [spring-petclinic](https://github.com/spring-projects/spring-petclinic) ?) but the code and comments seem to have diverged:
> A similar setup is provided for MySql ...

There's no `main/resources/application-mysql.properties` for example. I'm trying to run this using the Docker image [arey/springboot-petclinic](https://hub.docker.com/r/arey/springboot-petclinic/) adding in a likely properties file and hacking the classpath, but it's then failing with:

    java.sql.SQLException: Unable to load class: com.mysql.jdbc.Driver

Is it missing the [mysql-connector-java](https://mvnrepository.com/artifact/mysql/mysql-connector-java) artifact? I've been looking to add the driver post-hoc but having trouble finding something suitable in the Alpine repo, or shoehorning changes into the startup. I'd rather find a repackaging solution than start hacking the build myself ...

**Comment by arey** [comment]
Hi @hackery. Yes, this angularjs repository is a fork of the one you reference.
The MySQL support (code and documentation) has to be improve. If you are interested, I could work on it in the next days/weeks.

First, you may try to add the mysql-connecter-java artifact to the `spring-petclinic-server/pom.xml` 
```
    <dependency>
      <groupId>mysql</groupId>
      <artifactId>mysql-connector-java</artifactId>
      <scope>runtime</scope>
    </dependency>
```
You should rebuild the project and the Docker image.
I suppose there is also some lines to change in the `application.properties` file.

**Comment by arey** [comment]
Hi @hackery. I've reworked the MySQL configuration. Please have a look on my commit 
9a565ab
I hope this help you to use MySQL.

**Comment by hackery** [comment]
Yes, I'm interested, if you have time to spare on it. I can send you whatever feedback might be useful.
I've managed to bodge the mysql-connector into the runtime environment -
* Docker volume mapping of `mysql-connector-java-5.1.47-bin.jar` from dev.mysql.com
* some classpath/invocation trickery
* added an `application-mysql.properties`  file with some likely overrides for the db type, schema paths and db uri/credentials.

**Comment by arey** [comment]
I've published a `docker-compose.yml` file that start the Petclinic application with a MySQL database. The `readme.md` has been completed with a Docker usage documentation.
Finally, I've published a new Docker image into https://cloud.docker.com/u/arey/repository/docker/arey/springboot-petclinic
